resources/public/api/conf/2.0/retrieveSelfAssessmentBalanceAndTransactions.raml

```diff
@@ -25,50 +25,83 @@ description: This endpoint enables you to retrieve the overall liability broken
     type: string
     example: "3060013199"
     required: false
   fromDate:
-    description: The inclusive start date to return balances and transactions. (When onlyOpenItems has been set to false, either the date range (fromDate and toDate) OR docNumber should be supplied.)
+    description: |
+        The inclusive start date to return balances and transactions. The maximum date range between fromDate and toDate should not exceed 732 days.
+
+        (When onlyOpenItems has been set to false, either the date range (fromDate and toDate) or docNumber should be supplied.)
     type: string
     example: "2023-12-12"
     required: false
   toDate:
-    description: The inclusive end date to return balances and transactions. (When onlyOpenItems has been set to false, either the date range (fromDate and toDate) OR docNumber should be supplied.)
+    description: |
+        The inclusive end date to return balances and transactions. The maximum date range between fromDate and toDate should not exceed 732 days.
+
+        (When onlyOpenItems has been set to false, either the date range (fromDate and toDate) or docNumber should be supplied.)
     type: string
     example: "2024-12-12"
     required: false
   onlyOpenItems:
-    description: Limits the extraction to unpaid or not reversed charges. (When onlyOpenItems has been set to false, either the date range (fromDate and toDate) OR docNumber should be supplied.) Defaults to false
+    description: |
+        Limits the extraction to unpaid or not reversed charges.
+
+        (When onlyOpenItems has been set to false, either the date range (fromDate and toDate) or docNumber should be supplied.)
+
+        Defaults to false
     type: boolean
     example: false
     required: false
   includeLocks:
-    description: Include additional information related to claim and debt management. Defaults to false
+    description: |
+        Include additional information related to claim and debt management.
+
+        Defaults to false
     type: boolean
     example: true
     required: false
   calculateAccruedInterest:
-    description: "Calculate accrued interest. Accruing interest is the amount of interest calculated
-  a) To ‘today’s date (or in the case of a created statement, to the statement date)
-  b) On any overdue interest-bearing liability
-  No interest charge is created for an amount of accruing interest. \n An interest charge is only created when the related liability is paid in full."
+    description: |
+        Calculate accrued interest. Accruing interest is the amount of interest calculated
+
+        a) To today’s date (or in the case of a created statement, to the statement date)
+
+        b) On any overdue interest-bearing liability
+
+        No interest charge is created for an amount of accruing interest. \n An interest charge is only created when the related liability is paid in full.
```

Comment:
Could the second part (after the \n) be on the next line instead? 
(also there's a space after the \n, not sure if it would be rendered though)